mirrord/layer/src/file/ops.rs

```diff
@@ -159,13 +181,9 @@ fn close_remote_file_on_failure(fd: u64) -> Result<()> {
 /// [`OPEN_FILES`].
 #[mirrord_layer_macro::instrument(level = "trace", ret)]
 pub(crate) fn open(path: Detour<PathBuf>, open_options: OpenOptionsInternal) -> Detour<RawFd> {
-    let path = path?;
-
-    if path.is_relative() {
-        // Calls with non absolute paths are sent to libc::open.
-        Detour::Bypass(Bypass::RelativePath(path.clone()))?
-    };
+    let path = remap_path!(path?);
```

Comment:
remap should happen after relative

Reply:
But won't we want the ability to remap relative paths to absolute ones?

Reply:
Not at first step at least (haven't seen use case)